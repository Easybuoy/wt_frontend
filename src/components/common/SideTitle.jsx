import React from "react";
import { Heading, Text, Box } from "@chakra-ui/core";
import PropTypes from "prop-types";

function SideTitle({ heading, subheading }) {
  return (
    <Box textAlign="left" marginBottom="60px">
      <Heading size="lg">{heading}</Heading>
      {subheading ? <Text>{subheading}</Text> : null}
    </Box>
  );
}

//adding proptypes
SideTitle.propTypes = {
  heading: PropTypes.string.isRequired,
  subheading: PropTypes.string
};

export default SideTitle;
