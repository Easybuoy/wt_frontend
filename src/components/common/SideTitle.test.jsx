import React from "react";
import { shallow } from "enzyme";

import SideTitle from "./SideTitle";

describe("<SideTitle />", () => {
  let wrapper;

  const props = {
    heading: "",
    subheading: ""
  };

  beforeEach(() => {
    wrapper = shallow(<SideTitle {...props} />);
  });

  it("renders the Preview component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
