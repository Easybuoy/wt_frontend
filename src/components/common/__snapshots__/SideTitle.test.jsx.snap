// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideTitle /> renders the Preview component correctly 1`] = `
<Box
  marginBottom="60px"
  textAlign="left"
>
  <Heading />
</Box>
`;
